src/effects/VST/VSTEffect.h

```diff
@@ -101,6 +101,15 @@ struct VSTEffectSettings
 
 struct VSTEffectWrapper : public VSTEffectLink, public XMLTagHandler
 {
+   explicit VSTEffectWrapper(const PluginPath& path)
+      : mPath(path)
+   {}
+
+   ~VSTEffectWrapper()
```

Comment:
I just noticed in review today, the omission of `virtual` from this destructor, though there are other virtual functions.

It might look nicer to add it.  It's not in fact important, so long as there are never yet any unique or shared pointers to a VSTEffectWrapper.